api-specificatie/openapi.yaml

```diff
@@ -1,65 +1,58 @@
-          application/json:
-            schema:
-                $ref: '#/components/schemas/VolgindicatiePersoon'
+        Opvragen van alle actuele (actieve) volgindicaties van een abonnee. Volgindicaties met een einddatum in het verleden worden niet geleverd.
       responses:
-        '201':
+        200:
           description: "OK"
           headers:
             api-version:
               $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/headers/api_version"
             warning:
               $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/headers/warning"
           content:
-            application/json:
+            application/hal+json:
               schema:
-                $ref: '#/components/schemas/VolgindicatiePersoon'
-        '400':
-          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/400"
+                $ref: "#/components/schemas/VolgindicatiesHalCollectie"
         '401':
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/401"
         '403':
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/403"
-        '419':
-          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/409"
-        '415':
-          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/415"
+        '406':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/406"
         '500':
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/500"
         '503':
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/503"
         'default':
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/default"
       tags:
-      - volgindicatiepersonen
+      - Beheren volgindicaties
+  /volgindicaties/{burgerservicenummer}:
     get:
-      operationId: GetVolgindicatiePersonen
-      summary: "raadplegen alle volgindicaties"
+      operationId: GetVolgindicatie
       description: |
-        Opvragen lijst met burgerservicenummers waarop een abonnee een actuele volgindicatie heeft ingesteld.
+        Opvragen van een volgindicatie van een specifieke persoon.
+      summary: "Raadpleeg specifieke volgindicatie"
```

Comment:
Ik zou in ieder geval in de description expliciet opnemen dat het kan gaan om zowel actieve als verlopen volgindicaties.

Reply:
Dat lijkt me de standaard RESTfull werking, moet expliciet worden gemaakt dat het opvragen van een verlopen volgindicatie niet leidt tot een foutmelding?